dev_guide/migrating_applications/database_applications.adoc

```diff
@@ -0,0 +1,382 @@
+[[dev-guide-database-applications]]
+= Migrating Database Applications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This topic reviews how to migrate MySQL, PostgreSQL, and MongoDB database
+applications from OpenShift version 2 (v2) to OpenShift version 3 (v3).
+
+[[migrating-database-applications-supported-databases]]
+== Supported Databases
+
+[cols="2,2",options="header"]
+|===
+|v2 |v3
+
+|MongoDB: 2.4    |MongoDB: 2.4, 2.6
```

Comment:
I'm not sure but [we also have 3.2](https://github.com/sclorg/mongodb-container) (and 3.0 for migration purposes). Or it something else and shouldn't be mentioned here?